src/Microsoft.Win32.Registry/tests/RegistryKey/RegistryKey_OpenSubKey_str_rkpc.cs

```diff
@@ -50,7 +50,6 @@ public void NegativeTests()
         }
 
         [Fact]
```

Comment:
The bug is marked netfx-port-consider. So I assume, the test still fails on Desktop as we didn't port anything yet.
How come we can remove the attribute then? What am I missing?